demosplan/DemosPlanCoreBundle/Addon/AddonRoutingLoader.php

```diff
@@ -56,10 +58,18 @@ protected function configureRoute(Route $route, ReflectionClass $class, Reflecti
      */
     private function addControllers(mixed $addonInfo, RouteCollection $routeCollection): void
     {
-        $controllerDir = $addonInfo->getInstallPath().self::PATH_TO_CONTROLLERS_FROM_ADDONROOT;
-        $controllerPath = AddonPath::getRootPath($controllerDir);
-        if ('' !== $addonInfo->getInstallPath() && is_dir($controllerPath)) {
-            $routeCollection->addCollection($this->load($controllerPath));
+        // Without any installPath, we can't register any routes, so we can skip this step
+        if ('' === $addonInfo->getInstallPath()) {
+            return;
```

Comment:
Should we log this? Ideally, an exception should be thrown before we even end up here, but if we do, a silent return may not be easy to find if this fails in production.

Reply:
After looking at this a bit, I realized that `install_path` can never be an empty string as we set it like this:
''install_path' => 'addons/vendor/'.$addonName,'

Now the question becomes, do we want to add a better evaluation of this in this PR or would that be better handled in a follow-up improvement. If you don't mind, I would prefer to put that in a PR of its own as it has not really anything to do with this one.

Reply:
Id' suggest to delete this if clause because it never does anything it seems to do and do the error handling in another PR as it seems to be non-trivial in this case

Reply:
@graupnerdemos @AlexMenschoid could you please reach a decision so this PR could be merged? We need this one for a recent Addon

Reply:
We can add the evaluation at a later point, as it's not too critical at the moment.